snapcraft/__init__.py

```diff
@@ -377,6 +377,20 @@ def find_library(name):
     import ctypes.util
     ctypes.util.find_library = find_library
 
+
+def _get_version():
+    if _os.environ.get('SNAP_NAME') == 'snapcraft':
+        return _os.environ["SNAP_VERSION"]
+    try:
+        return pkg_resources.require('snapcraft')[0].version
+    except pkg_resources.DistributionNotFound:
+        return 'devel'
+
+
+# Set this early so that the circular imports aren't too painful
+__version__ = _get_version()
```

Comment:
Another small detail. Could you please this variable right before the _get_version definition?
I'm trying to get our source code to be readable from top to bottom, and those details help.